10-JavaScript/exercises/objects-worksheet.md

```diff
@@ -10,7 +11,7 @@ console.log(pen.ink);
 2. What does the following code print to the console?
 ```javascript
 let bottle = {
-  contents: function () { return "some fine juice" },
+  contents: () => "some fine juice",
```

Comment:
Just an observation, arrow functions typically make poor methods.  This one doesn't matter since it doesn't use `this`, but we don't want students imitating it.

Reply:
ok - i'll leave it now for a demonstration but noted.